### PR TITLE
add k8s config

### PR DESCRIPTION
**Pull request by idcrosby**
This is a *temporary* solution to have a Kubernetes specific dev config.

### CODE REVIEW

**Comment by bobvanluijt** [comment]
> This is a _temporary_ solution to have a Kubernetes specific dev config.

Should it be merged though?

**Comment by moretea** [comment]
It won't hurt, would it?

What exactly do you need @idcrosby ? An easier way to configure the hostname of Janus?
We could add an env variable that would override the settings from the configuration file.

File: tools/dev/config.json [hunk 1]
```diff
@@ -89,6 +89,39 @@
...
+        },
+        "local_schema": {
+          "state_dir": "."
+        }
+      },
+      "contextionary": {
+        "knn_file" : "/contextionary/example.knn",
+        "idx_file" : "/contextionary/example.idx"
+      },
+      "schemas": {
```
Comment (moretea) [inline tools/dev/config.json]: These are not needed anymore; I'll open a PR soon to remove them all.

**Comment by idcrosby** [comment]
@moretea yes, all that's needed is a way to pass in the janus hostname. This was a temp quick fix for that (without me having to dig in to the weaviate code). If we could add an env var that would be better.

**Comment by bobvanluijt** [comment]
hi @idcrosby and @moretea, please let me know when this PR is ready to merge.

**Comment by idcrosby** [comment]
In my opinion this can be merged.

**Pull request merged**